src/status_im/contexts/onboarding/generating_keys/view.cljs

```diff
@@ -123,4 +103,4 @@
      (sequence-animation generate-keys-opacity saving-keys-opacity keys-saved-opacity)
```

Comment:
we should also be deleting these opacities no? they're unused now.
There was also some mechanism @briansztamfater had used to check the status ok keys generated to block the transition until it was ready. Might be good to revisit "when" the keys are generated in the onboarding flow so this page transitions faster too. I discussed this before with @cammellos, basic idea is to actually do the background generation once we get the minimum amount of required data in the onboarding flow so this page is just for "show" and not indicative of what is actually happening at that time.

Reply:
@J-Son89 these are still being used by the screen title. I only updated the illustration for this reason unless we also just want to show a static title

Reply:
Thanks!

Reply:
I did not remove them instead just refactored the code to have them in the title component which is the only place they are being used.